src/fx/compute-options.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * compute-options
+ * @author Deux Huit Huit
+ * 
+ * Translate xml attributes into a js object with camelcase key properties
+ * 
+ * Example:
+ * 
+ * <div data-mykey-foo-foo="cool" data-my-key-bar="true" data-my-key-baz="10" />
+ * 
+ * will translate into
+ * 
+ * {
+ *    fooFoo: 'cool',
+ *    bar: true,
+ *    baz: 10
+ * }
+ * 
+ * Usage:
+ * 
+ * App.fx.notify('computeOptions', {element: $('div'), key: 'mykey'}, function (index, options) {});
+ */
+(function ($, undefined) {
+
+	'use strict';
+
+	var computeOptions = function (key, options) {
+		var opts = {};
+
+		try {
```

Comment:
Sorry my comment was not clear.
What I meant was to put only the creation of the RegExp in the try catch block.
Also, put that in a seperate function, because try/catch is a deop flag (function with try/catch can not be optimized by V8)

Reply:
@nitriques done.